### PR TITLE
Remove dependency on accretion

### PR DESCRIPTION
**Pull request by acron0**
Some large changes to the workflow here....please look at https://docs.google.com/drawings/d/1FwdIzHT06v0Rnq1pSCv6iGL8gPJoC5uq7EGp750EtOw/edit

### CODE REVIEW

File: src/witan/models/dem/ccm/core/projection_loop.clj [hunk 1]
```diff
@@ -12,48 +12,67 @@
...
+  (let [loop-year (m-utils/get-last-year population)]
+    (>= loop-year last-proj-year)))
 
-(defworkflowfn prepare-inputs
+(defworkflowfn prepare-inputs-1-0-0
   "Step happening before the projection loop.
    Takes in the historic population and outputs the latest year and
    the population for that year which will be updated within the
    projection loop. Also outputs the the population that will ultimately
    contain the historic population and the population projections."
```
Comment (sunnytownsend) [inline src/witan/models/dem/ccm/core/projection_loop.clj:33]: doc strings need updating generally but have created an epic which includes this

File: test/witan/models/run_models_test.clj [hunk 2]
```diff
@@ -80,29 +80,29 @@
...
-      (is proj-bristol-2015)
-      (is (= (:shape proj-bristol-2015) (:shape r-proj-bristol-2015)))
-      (is (every? #(fp-equals? (wds/subset-ds joined-ds-2015 :rows % :cols :popn)
-                               (wds/subset-ds joined-ds-2015 :rows % :cols :popn-r) 0.0000000001)
-                  (range (first (:shape joined-ds-2015)))))
-      (is (= (:shape proj-bristol-2040) (:shape r-proj-bristol-2040)))
-      (is (every? #(fp-equals? (wds/subset-ds joined-ds-2040 :rows % :cols :popn)
-                               (wds/subset-ds joined-ds-2040 :rows % :cols :popn-r) 0.0000000001)
-                  (range (first (:shape joined-ds-2040))))))))
+#_(deftest run-workspace-test
```
Comment (tcoupland) [inline test/witan/models/run_models_test.clj:83]: Seems like a pretty big test to be commented out :)

Comment (acron0) [reply test/witan/models/run_models_test.clj:83]: Instruction is to not maintain the demo tool anymore, which this test is for

Comment (tcoupland) [reply test/witan/models/run_models_test.clj:83]: cool

**Comment by tcoupland** [comment]
can't see anything out of wack.

:shipit:

**Pull request merged**